Add details about flag mismatch (#39)
k3s-io/docs: docs/installation/configuration.md

For details on configuring the K3s agent, see [Agent Configuration.](reference/agent-config.md)  
You can also use the `--help` flag to see a list of all available options.

> :::info Matching Flags
> 
> It is important to match critical flags on your server/agent installations. For example, if you use the flag
> `--disable servicelb` or `--cluster-cidr=10.42.0.0/16` on your master node, but don't set it on your other installs,
> your nodes will fail to join. They may print errors such as 
> `starting kubernetes: preparing server: failed to validate server configuration: critical configuration value mismatch`.
> 
> 
> 

Reviewer: ```suggestion
:::info Matching Flags
It is important to match critical flags on your server/agent installations. For example, if you use the flag
`--disable servicelb` or `--cluster-cidr=10.42.0.0/16` on your master node, but don't set it on other server nodes, the nodes will fail to join. They will print errors with:
`failed to validate server configuration: critical configuration value mismatch.`
:::
```

Reviewer: You have to close the admonition, try viewing your changes using `yarn start`. Currently it results in 
![image](https://user-images.githubusercontent.com/11727736/200045790-46b54a79-3d8d-405c-bf98-a16c6bf8ef2d.png)
